LuaRules/Gadgets/exp_nuke_effect_chooser.lua

```diff
@@ -42,8 +42,10 @@ function gadget:Explosion_GetWantedWeaponDef()
 end
 
 function gadget:Explosion(weaponID, px, py, pz, ownerID)
-	if (nux[weaponID] and py-GetGroundHeight(px,pz)>100) then
-		return true
+	if (nux[weaponID] and py-GetGroundHeight(px,pz)>200) then
```

Comment:
Funnel shield is pretty tall (weaponDef says 550 radius at 45 emitHeight) so 200 can still result in the interception gfx:
![image](https://user-images.githubusercontent.com/2573076/54887542-749bba80-4e94-11e9-8b0f-dc1c25a7ed2c.png)


```suggestion
	if (nux[weaponID] and py-GetGroundHeight(px,pz)>600) then
```

Reply:
I think 600 is too much - at some point a Funnelweb interception actually *is* an interception - even if partial one, with a reduced rather than absent ground aoe radius.

Reply:
Alright. Another question is sea: `GetGroundHeight` returns negative values there so at deep enough sea you can get the interception fx  when nuking ships:
![image](https://user-images.githubusercontent.com/2573076/54944297-7799b800-4f33-11e9-97e1-259aac7d7b36.png)
```suggestion
	if (nux[weaponID] and py-math.max(0, GetGroundHeight(px,pz))>200) then
```

Reply:
Good catch!